corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -35,7 +35,7 @@
 
 @register.simple_tag
 def render_form_xml(form):
-    xml = form.get_xml()
+    xml = form.get_xml().encode('utf-8')
```

Comment:
i meant the python type. @millerdev will know the answer. i think as long as we're sure it's an ascii or utf-8 encoded string I think it's fine, but to be safe you can also check the python type. just want to avoid double-encoding it